app/models/model.rb

```diff
@@ -0,0 +1,6 @@
+class Model < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+end
```

Comment:
What is it about?
It's not used anywhere. Please, remove it.

Reply:
Done. Thank you Maelfosso.

Reply:
:100:  done!!